tools/dmitool/git_merge_installer.bat

```diff
@@ -1,6 +1,6 @@
-@echo off

```

Comment:
this file literally hasn't changed at all and Github recognises that but still includes it here